server/src/models/setlist.model.ts

```diff
@@ -0,0 +1,25 @@
+import { Schema, model, models, Types } from 'mongoose';
+
+interface ISetlist {
+  name: string;
+  date: string;
+  createdBy: string;
```

Comment:
this should be Types.ObjectId because a user (with a userId) will create this setlist